sdk/storage/azure-storage-files-datalake/CMakeLists.txt

```diff
@@ -39,13 +39,14 @@ set(
     inc/azure/storage/files/datalake/datalake_sas_builder.hpp
     inc/azure/storage/files/datalake/datalake_service_client.hpp
     inc/azure/storage/files/datalake/dll_import_export.hpp
-    inc/azure/storage/files/datalake/internal/datalake_utilities.hpp
-    inc/azure/storage/files/datalake/internal/datalake_constants.hpp
+
```

Comment:
nit: delete extra new line

Reply:
https://github.com/Azure/azure-sdk-for-cpp/pull/2435